pages/_app.js

```diff
@@ -10,9 +10,12 @@ import OnboardJS from '../src/contextProvider/Web3Provider'
 
 import '../styles/globals.css'
 import '../src/components/richImageUploader/quill.imageUploader.css'
+import { OverlayMaintenace } from '../src/components/overlays/Maintenance'
+import { useState } from 'react'
 
 function MyApp({ Component, pageProps }) {
   const apolloClient = useApollo(pageProps)
+  const [showMaintenance, setShowMaintenance] = useState(true)
```

Comment:
Looking great man!
Should we set this from an environment variable instead?

Reply:
yeah I, Think it's better to set it from env.
Thanks @mateodaza